### PR TITLE
Updated CommandLineParser to latest version (2.8.0)

### PR DESCRIPTION
**Pull request by mertyeter**
I replaced source codes with up-to-date nuget package and did necessary changes related to the latest version.

### CODE REVIEW

File: src/SQLQueryStress/CommandLineOptions.cs [hunk 1]
```diff
@@ -1,47 +1,27 @@
...
+        public string DbServer { get; set; } = string.Empty;
 
-        [Option("t", null,
+        [Option('t', "threads",
                 HelpText = "Number of threads in unattended mode")]
-        public int NumberOfThreads = -1;
-
-        [HelpOption("?", null,
-                HelpText = "Display this help screen")]
-        public string GetUsage()
```
Comment (ErikEJ) [inline src/SQLQueryStress/CommandLineOptions.cs:33]: What about the nice help screen??

File: src/SQLQueryStress/Program.cs [hunk 2]
```diff
@@ -14,47 +15,72 @@ internal static class Program
...
-            else
+            else if (errors.IsHelp())
             {
-                var f = new Form1(options)
+                var helpText = HelpText.AutoBuild(result, help =>
                 {
-                    StartPosition = FormStartPosition.CenterScreen
-                };
-                Application.Run(f);
+                    help.AdditionalNewLineAfterOption = true;
```
Comment (ErikEJ) [inline src/SQLQueryStress/Program.cs:52]: Can we get the version and copyright information back also?

File: src/SQLQueryStress/CommandLineOptions.cs (hunk 1 above)
Comment (mertyeter) [reply src/SQLQueryStress/CommandLineOptions.cs:33]: HelpOption was removed and replaced with HelpText.AutoBuild. Please see DisplayHelp method on Program.cs

**Comment by ErikEJ** [comment]
Excellent, nice to get rid of all these files!

File: src/SQLQueryStress/Program.cs (hunk 2 above)
Comment (mertyeter) [reply src/SQLQueryStress/Program.cs:52]: They were also moved to HelpText.AutoBuild related to CommandLineParser changes.

Comment (ErikEJ) [reply src/SQLQueryStress/Program.cs:52]: I dont see it, but maybe you can share a screen  shot of the help screen?

Comment (mertyeter) [reply src/SQLQueryStress/Program.cs:52]: image

**Pull request merged**